tool/plugin/test/plugin_test.dart

```diff
@@ -178,6 +218,8 @@ BuildCommandRunner makeTestRunner() {
 class TestBuildCommand extends BuildCommand {
   TestBuildCommand(runner) : super(runner);
 
+  bool get isTesting => true;
```

Comment:
We probably should've defined a mixin for testing. Something to keep in mind for later.